index.html

```diff
@@ -208,5 +208,30 @@ <h4>Skills</h4>
                 </ul>
                 <a href="#"><h5>tyaps002@gmail.com</h5></a>
             </section>
+            
+            <section class="form-survey">
+                <form action="https://formspree.io/f/mpzbbknz"  method="POST" class="form">
+                    <p>I'm always interested in hearing about new projects, 
+                        so if you'd like to chat please get in touch.</p>
+                    <div class="inputs">
+                        <input type="text" id="name" placeholder="Full name">
+                        <input type="email" id="email" placeholder="Email address">
+                        <textarea name="text" id="message" placeholder="Enter text here" cols="30" rows="10"></textarea>
+                    </div>
+                        <button type="button" class="submit">Get in touch</button>
+                        <div id="form-media">
+                        <ul>
+                            <li><a href=""><img src="images/twitter.svg" alt="my twitter profil"></a></li>
+                                <li><a href=""><img src="images/linked.svg" alt="connect on linkedin"></a></li>
+                                <li><a href=""><img src="images/medium.svg" alt="check medium"></a></li>
+                                <li><a href=""><img src="images/github.svg" alt="github profile"></a></li>
+                                <li><a href=""><img src="images/angel.svg" alt="angel list"></a></li>
+                        </ul>
```

Comment:
- [ ] for semantic purpose and clean code, kindly remove the social icons from the `form tag`

Reply:
I would have loved to adjust this but unfortunately, it will disrupt the rest of my project because I have the social icons in other part of my projects